src/commands/run.ts

```diff
@@ -0,0 +1,62 @@
+import ICommand from '@/library/interfaces/iCommand';
+import Languages from '@/library/languages';
+import axios, { AxiosResponse } from 'axios';
+import { Message } from 'discord.js';
+import { parse } from 'querystring';
+
+const languages = new Languages();
+
+// Hack for implementing with static properties/methods
+let Run: ICommand;
+export default Run = class {
+
+  /* istanbul ignore next */
+  static get description(): string {
+    return 'Executes provided code using a LanguageRunner';
+  }
+
+  public static execute(args: string[], msg: Message) {
+
+    let parseResponse;
+
+    try {
+      parseResponse = this.parseCode(msg.content);
+      if (!languages.get(parseResponse.language)) {
+        msg.channel.send(`Unknown language: ${parseResponse.language}`);
+        return;
+      }
+    } catch {
```

Comment:
You might want to display the caught error in the message to give additional context, and since one of the possible errors it will catch will be: 

https://github.com/osu-cascades/hackbot/pull/124/files#diff-f3956dc0c020537b000490fd556f9331R58

```
throw new Error(`Unable to extract code from ${messageText}`);
```

Reply:
Done